Formula/minishift.rb

```diff
@@ -0,0 +1,41 @@
+require "language/go"
+
+class Minishift < Formula
+  desc "Runs a single-node OpenShift cluster inside a VM"
+  homepage "https://minishift.io/"
+  url "https://github.com/minishift/minishift/archive/v1.1.0.tar.gz"
+  version "1.1.0"
```

Comment:
Yeah. See the audit error about its being redundant since it was already determined to be the same value from the URL.